src/debugAdapter/contracts/contractHelpers.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Consensys Software Inc. All rights reserved.
+// Licensed under the MIT license.
+
+import Compiler from '@truffle/workflow-compile';
+import Config from '@truffle/config';
+import {Compilations} from '@truffle/codec';
+import path from 'path';
+import {Source} from '@truffle/compile-common';
+
+export namespace ContractHelper {
```

Comment:
everything looks good here, do we want to keep using namespaces as its not really a modern idiom of TS (most linters suggest to stop using namespaces and use exports instead). This is up for discussion I suppose.

Reply:
Hey @michaeljohnbennett,

Thank you for review it. Yeah, i just kept the structure, but we can change that. I'll convert it to export way.

Cheers!

Reply:
I would stop using `namespace`s in favor of `export`s. I don't think `namespace`s add a valuable abstraction mechanism that is not already provided by modules/`export`s.